### ISSUE TITLE
Autocomplete using Windows high contrast mode (WHCM)

### ISSUE THREAD

**Issue by danielelder** [alphagov/accessible-autocomplete#483]
When using Windows high contrast mode (High Contrast Blank theme) the focus and hover style of items in the list are displayed as white text on a black background, i.e. no difference to the unfocused default state..

This would prevent a keyboard only user using high contrast mode from being able to know which item in the list they are selecting.

A possible solution would be to apply some high contrast styles to the css to set the colour to the [system colour](https://www.w3.org/TR/css-color-4/#css-system-colors) when high contrast mode is enabled, for example:

```
/* IE 11 and Edge 17 and below */
@media (-ms-high-contrast: active) {
  .autocomplete__option--focused,
  .autocomplete__option:hover {
    -ms-high-contrast-adjust: none;
    background-color: highlight;
    color: highlightText;  
  }
}

/* Edge 18+ */
@media (forced-colors: active) {
  .autocomplete__option--focused,
  .autocomplete__option:hover {
    -ms-high-contrast-adjust: none;
    background-color: highlight;
    color: highlightText;  
  }
}
```
Screenshots of current styling:

<img width="582" alt="autocomplete-current-whcm" src="https://user-images.githubusercontent.com/16639519/117160725-469a3b80-adb9-11eb-85c4-56f60fd3e76e.png">

Screen using high contrast media query with system colours:

<img width="589" alt="autocomplete-whcm-mq" src="https://user-images.githubusercontent.com/16639519/117160857-603b8300-adb9-11eb-996e-be5414421bcd.png">

**Comment by vanitabarrett** [comment]
I think this is a duplicate of https://github.com/alphagov/accessible-autocomplete/issues/369 and should be fixed by https://github.com/alphagov/accessible-autocomplete/pull/512
